src/realm/parser/generated/query_flex.cpp

```diff
@@ -101,6 +101,7 @@ typedef int16_t flex_int16_t;
 typedef uint16_t flex_uint16_t;
 typedef int32_t flex_int32_t;
 typedef uint32_t flex_uint32_t;
+typedef uint64_t flex_uint64_t;
```

Comment:
Can you remove all the changes from query_flex.cpp/query_bison.cpp? I think there's an issue with the version of bison/flex installed on MacOS via homebrew and we don't want to check in these auto-generated changes yet.